CS-FUNdamentals.rb

```diff
@@ -0,0 +1,54 @@
+
+def print_array(array)
+  print "["
+  i = 0
+  array.length.times do
+  if i == array.length-1
+  print array[i]
+  print  "]"
+  else
+  print array[i]
+  print ", "
+  i += 1
+  end
+  end
+end
```

Comment:
You need to watch your indentation. It's super important to the readability of your code.